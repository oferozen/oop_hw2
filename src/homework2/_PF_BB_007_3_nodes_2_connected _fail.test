#syntax:
# CreateNode n w
# CreateGraph n
# AddNode g n
# AddEdge A n3 n1 
# FindPath g n1 n2 -> n3 n4

CreateGraph g
CreateNode n1 1
CreateNode n2 1
CreateNode n3 1
AddNode g n1
AddNode g n2
AddNode g n3
AddEdge g n1 n2
AddEdge g n2 n1
AddEdge g n1 n1
AddEdge g n2 n2
FindPath g n1 -> n3
FindPath g n2 -> n3