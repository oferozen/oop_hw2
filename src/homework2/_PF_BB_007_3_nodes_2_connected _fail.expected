#syntax:
# CreateNode n w
# CreateGraph n
# AddNode g n
# AddEdge A n3 n1 
# FindPath g n1 n2 -> n3 n4

created graph g
created node n1 with cost 1
created node n2 with cost 1
created node n3 with cost 1
added node n1 to g
added node n2 to g
added node n3 to g
added edge from n1 to n2 in g
added edge from n2 to n1 in g
added edge from n1 to n1 in g
added edge from n2 to n2 in g
shortest path in g:
shortest path in g:
